### DISCUSSION TITLE
Not Seeing CodeQL in Status Checks for Branch Protection Rules

### DISCUSSION THREAD

**Discussion by SPais360Insights** [community/community#69354] [Code Security]
Hi there,
Looking for some help as I have setup CodeQL analysis for Code Scanning and wanted to prevent merges if there are CodeQL warnings. I understand you can do that via Status Checks for Branch Protection Rules.
CodeQL shows up in some of our Node and Reach projects but not others  and I am baffled why.
![image](https://github.com/community/community/assets/61429864/9fc6ed5f-285d-4bfd-9ffb-542964dfeb49)

**Comment by jketema** [comment]
Hi @SPais360Insights,

Thanks for your question. The CodeQL workflow needs to have run recently on the repository before it's picked up. Is this the case?

**Reply by SPais360Insights** [reply]
Hmm, that was what I suspected as well. How recently should it have run?

**Reply by jketema** [reply]
Not sure, but you can always start the workflow manually to ensure it has run recently.

**Reply by SPais360Insights** [reply]
I can confirm that worked for 99% of the cases! Thanks!
